grype/presenter/models/vulnerability.go

```diff
@@ -5,71 +5,61 @@ import (
 	"github.com/anchore/grype/grype/vulnerability"
 )
 
-type Cvss struct {
-	BaseScore           float64  `json:"baseScore"`
-	ExploitabilityScore *float64 `json:"exploitabilityScore,omitempty"`
-	ImpactScore         *float64 `json:"impactScore,omitempty"`
-	Vector              string   `json:"vector"`
-}
-
 type Vulnerability struct {
 	ID             string   `json:"id"`
 	Severity       string   `json:"severity,omitempty"`
 	Links          []string `json:"links,omitempty"`
 	Description    string   `json:"description,omitempty"`
-	CvssV2         *Cvss    `json:"cvssV2,omitempty"`
-	CvssV3         *Cvss    `json:"cvssV3,omitempty"`
+	Cvss           []Cvss   `json:"cvss,omitempty"`
 	FixedInVersion string   `json:"fixedInVersion,omitempty"`
 }
 
+type Cvss struct {
+	Version string      `json:"version"`
+	Vector  string      `json:"vector"`
+	Metrics CvssMetrics `json:"metrics"`
+}
+
+type CvssMetrics struct {
+	BaseScore           float64  `json:"base_score"`
```

Comment:
@alfredodeza Just a reminder that we use `camelCase` for JSON field names (words are demarcated by capital letters, first word is lowercase)

Reply:
This got updated